barretenberg/cpp/src/barretenberg/vm/avm/trace/helper.cpp

```diff
@@ -122,6 +124,11 @@ std::string to_name(AvmError error)
     }
 }
 
+bool is_valid(AvmError error)
```

Comment:
Might be better to rename to "is_ok / has_error / is_failure / no_error" or something like that.

Reply:
I renamed to is_ok()